backend/apps/search/documents.py

```diff
@@ -55,5 +57,26 @@ class Django:
             "created_at",
             "updated_at",
         ]
+        related_models = ['startup']
+
```

Comment:
In django-elasticsearch-dsl, the list should contain model classes, not rows.

It should be like this:

    related_models = [StartupProfile]

Otherwise, reindexing based on signals will not work (data will not be updated when the startup changes).